### PR TITLE
v 2.5.0

### PR DESCRIPTION
**Pull request by bartoszwalacik**

### CODE REVIEW

File: documentation/domain-configuration/index.md [hunk 1]
```diff
@@ -217,16 +217,22 @@ Three **JPA** Class level annotations are interpreted as synonyms of JaVers anno
...
 * [`@DiffIgnore`]({{ site.javadoc_url }}index.html?org/javers/core/metamodel/annotation/DiffIgnore.html)
   &mdash;
-  declares a property as ignored by the diff engine.
+  declares a property as ignored by JaVers.
 
+* [`@ShallowReference`]({{ site.javadoc_url }}index.html?org/javers/core/metamodel/annotation/ShallowReference.html)
+    &mdash;
+  declares a property as `ShallowReference`.
+  Can be used only for Entity type properties.
+  All properties of a target Entity instance, instead of Id, are ignored.
```
Comment (mwesolowski) [inline documentation/domain-configuration/index.md]: raczej except from zamiast instead of

File: documentation/features/index.md [hunk 2]
```diff
@@ -89,6 +89,17 @@ with minimal mapping configuration effort
 
 <h2 id="release-notes">Release notes</h2>
 
+### 2.5.0
+released on 2016-10-25 <br/>
+
+* [412](https://github.com/javers/javers/issues/412)
```
Comment (mwesolowski) [inline documentation/features/index.md]: [#412] zamiast [412]

File: documentation/domain-configuration/index.md (hunk 1 above)
Comment (bartoszwalacik) [inline documentation/domain-configuration/index.md]: thnx

File: documentation/features/index.md (hunk 2 above)
Comment (bartoszwalacik) [inline documentation/features/index.md]: ok

**Pull request merged**